tests/integration_python/conftest.py

```diff
@@ -175,7 +175,7 @@ def tmp_pixi_workspace(tmp_path_factory: pytest.TempPathFactory) -> Path:
 
     pixi_config = """
 # Reset to defaults
-default-channels = ["conda-forge"]
+default-channels = ["https://prefix.dev/conda-forge"]
```

Comment:
That one I'd like to keep.

That file is set to our defaults, and it only affects `pixi init`

Reply:
Fixed